azure-pipelines.yml

```diff
@@ -28,17 +28,49 @@ jobs:
       export CXX='g++-8'
       mkdir -p $(Build.Repository.LocalPath)/build
       cd $(Build.Repository.LocalPath)/build
-      cmake --version 
+      cmake --version
       cmake .. -DPYTHON_EXECUTABLE=$(which python3.7) -DCMAKE_INSTALL_PREFIX=$HOME/nmodl -DCMAKE_BUILD_TYPE=Release
       make -j 2
       if [ $? -ne 0 ]
       then
         make VERBOSE=1
       fi
+      make install
       env CTEST_OUTPUT_ON_FAILURE=1 make test
     env:
       CMAKE_PKG: 'cmake-3.10.2-Linux-x86_64'
-    displayName: 'Build and Run Tests'
+    displayName: 'Build and Run Unit Tests'
+#  - script: |
+#      export PATH=$(pwd)/$CMAKE_PKG/bin:/home/vsts/.local/bin:$PATH
+#      export CXX='g++-8'
+#      git clone --recursive https://github.com/BlueBrain/CoreNeuron.git
+#      mkdir CoreNeuron/build
+#      cd CoreNeuron/build
+#      cmake --version
+#      cmake .. -DCORENRN_ENABLE_MPI=OFF -DCORENRN_ENABLE_NMODL=ON -DCORENRN_NMODL_DIR=$HOME/nmodl -Dnmodl_PYTHONPATH=$HOME/nmodl/lib -DCORENRN_NMODL_FLAGS="sympy --analytic"
+#      make -j 2
+#      if [ $? -ne 0 ]
+#      then
+#        make VERBOSE=1
+#      fi
+#      ./bin/nrnivmodl-core $(Build.Repository.LocalPath)/test/integration/mod
+#    env:
+#      CMAKE_PKG: 'cmake-3.10.2-Linux-x86_64'
+#    displayName: 'Build and Run Integration Tests'
+  - script: |
+      export PATH=$(pwd)/$CMAKE_PKG/bin:/home/vsts/.local/bin:$PATH
+      export CXX='g++-8'
+      python3.7 -m pip install -U virtualenv scikit-build
+      python3.7 setup.py sdist bdist_wheel
+      rm -rf venv
```

Comment:
Also remove here build directories created by setup.py. This way we are sure that there is nothing from build/install dir of cmake is being used.